eirslett/storybook-builder-vite
# Upgrade to storybook/csf-tools for MDX processing

## What I did

Use `@storybook/csf-tools` for MDX compilation to avoid large dependency on `@storybook/addon-docs`, per request by @TheComputerM 

## How to test

I'm not sure. Basically the latest RC of Storybook breaks vite-builder-storybook because addon-docs dist file no longer exists in RC.3. So we need to publish the builder and then update the examples to use 0.0.8.
